airflow/api_connexion/endpoints/log_endpoint.py

```diff
@@ -15,12 +15,85 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# TODO(mik-laj): We have to implement it.
-#     Do you want to help? Please look at: https://github.com/apache/airflow/issues/8135
+from accept_types import get_best_match
```

Comment:
FYI there is a build-in function from flask:
https://github.com/apache/airflow/blob/8446526882a5798d838224b7566d5349111fd0d0/airflow/api_connexion/endpoints/config_endpoint.py#L30

Reply:
Thanks for this!